include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -1114,7 +1114,12 @@ def CallOp : TT_Op<"call", [CallOpInterface, /*MemRefsNormalizable, */DeclareOpI
     MutableOperandRange getArgOperandsMutable() {
       return getOperandsMutable();
     }
-
+    Attribute removeArgAttrsAttr() { return nullptr; }
+    Attribute removeResAttrsAttr() { return nullptr; }
+    ArrayAttr getArgAttrsAttr() { return nullptr; }
+    ArrayAttr getResAttrsAttr() { return nullptr; }
+    void setArgAttrsAttr(ArrayAttr) { return; }
+    void setResAttrsAttr(ArrayAttr) { return; }
```

Comment:
I think these aren't actually needed? I know this is to integrate https://github.com/llvm/llvm-project/pull/123176 but these will already be auto-generated I believe. For example bump that also integrates that PR doesn't require such a change https://github.com/triton-lang/triton/pull/5969.

Reply:
You are right. I didn't notice that https://github.com/triton-lang/triton/commit/ff77e9844a3047751895540fbe8e7779bde6b061 did this the right way.

Removed.